pkg/interactive/interactive.go

```diff
@@ -38,6 +40,30 @@ type Input struct {
 	Validators []Validator
 }
 
+func (i *Input) LenOptions() int {
+	if i == nil {
```

Comment:
is this possible?

Reply:
😵‍💫  it's been a while since writing some code. lol. hope you forgive the silly mistake. 😄